client/my-sites/domains/map-domain/index.jsx

```diff
@@ -115,6 +121,21 @@ export class MapDomain extends Component {
 				( error ) => this.setState( { errorMessage: error.message } )
```

Comment:
Does this special VIP error handling code also need to set `isBusyMapping: false`?

Reply:
Ah, yeah, was going to update that, but forgot 🤦 It's not a big issue, cause according to the REST API stats, this flow has not been used since November 2020 (and we plan to remove it completely as a follow-up here).
I'll update it 🙇‍♂️